EncoreTickets.SDK/Utilities/CommonModels/Extensions/PriceWithCurrencyExtension.cs

```diff
@@ -5,6 +5,22 @@ namespace EncoreTickets.SDK.Utilities.CommonModels.Extensions
 {
     public static class PriceWithCurrencyExtension
     {
+        /// <summary>
+        /// Returns the price in string format.
+        /// </summary>
+        /// <param name="price"></param>
+        /// <returns>The user-friendly string with value and currency.</returns>
+        public static string ToStringFormat<T>(this T price)
+            where T : IPriceWithCurrency
+        {
+            var powerToConvertToRealValue = Math.Pow(10, price.DecimalPlaces ?? 2);
+            var realValue = price.Value / powerToConvertToRealValue;
+            var valueAsStr = realValue.HasValue && realValue - Math.Truncate(realValue.Value) > 0D
+                ? $"{realValue:F20}".TrimEnd('0')
```

Comment:
Why would you need to remove trailing zeros from a price? Most of the time it makes sense to write $45.00 and $45.50 instead of $45 and $45.5. Also it looks like you are treating money as double instead of decimal which is not a good idea